fix (docs): renamed platform-related terms (#2610)
commercetools/merchant-center-application-kit: website/src/content/concepts/oauth-scopes-and-user-permissions.mdx

# OAuth Scopes

> Custom Applications can potentially use any of the available Composable Commerce APIs. This is very important and one of the reasons for using Custom Applications in the first place: to be able to extend and improve functionalities of the commercetools Composable Commerce and Merchant Center.

Reviewer: ```suggestion
Custom Applications can use any of the available Composable Commerce APIs. This allows you to extend the functionality and build custom features for any of the Composable Commerce APIs and the Merchant Center.
```
